### ISSUE TITLE
How about meet XDG standards, move config, cache, data dir to XDG dirs?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#148]
```
http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html

although pino does put config files into ~/.config. but this method is not 
good enough. 

pino set the path staticly and didn't distinguish config files, cache 
files and data files. i suggest to put config files to $XDG_CONFIG_HOME/
(default is ~/.config)/pino, cache files to $XDG_CACHE_HOME(~/.cache)/
pino, data files to $XDG_DATA_HOME(~/.local/share)/pino.

it has many advantages, e.g. users can easily delete what they don't want, 
like cache files, just need to delete ~/.cache, and all applications' 
cache files will be deleted.

it won't need change a lot. glib2 provides g_get_user_config_dir() to get 
$XDG_CONFIG_HOME, python have pyxdg. really tiny changes.
```

Original issue reported on code.google.com by `jarry...@gmail.com` on 26 Mar 2010 at 4:28

**Comment by GoogleCodeExporter** [comment]
```
Here's a patch.

Relevant documentation is at:
http://references.valadoc.org/glib-2.0/GLib.Environment.html

```

Original comment by `tar...@gmail.com` on 27 Mar 2010 at 1:05

Attachments:
- [pino-xdg-dirs.patch](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-148/comment-1/pino-xdg-dirs.patch)

**Comment by GoogleCodeExporter** [comment]
```
Closed by 5a28e53997 revision. Thanks.
```

Original comment by `tro...@gmail.com` on 27 Mar 2010 at 9:19
- Changed state: **Fixed**
